pages/learn/more/examples/projects.md

```diff
@@ -201,6 +201,12 @@ visualization at [123k.de](http://123k.de).
 A dashboard which is visualizing the [Uptime Robot API](https://uptimerobot.com/) on a Raspberry Pi, deployed via balenaCloud.
 This dashboard can be used on TVs or on monitors.
 
+### Balena ADS-B Multi-Service Flight Tracker
+
+[Repository][balena-ads-b] by [ketilmo][ketilmo]
+
+This is project allows you to log humidity and temperature data from a simple sensor and stores that data in a [firebase][firebase] backend.
```

Comment:
This description isn't correct!

Reply:
Sorry @chrisys, my mistake. Copy-pasting gone wrong. I will fix this and get back to you.

Reply:
@chrisys I have corrected the project description now. Feel free to have a look again.